application-logging/esm-logs-in-context/env.example

```diff
@@ -0,0 +1,8 @@
+NEW_RELIC_LICENSE_KEY=<your key>
+NEW_RELIC_APP_NAME=logs-in-context-esm-demo
+NEW_RELIC_LOG=stdout
+
+# For New Relic devs you can use staging
+# NEW_RELIC_HOST=staging-collector.newrelic.com
+# NEW_RELIC_LOG_HOST=https://staging-log-api.newrelic.com/log/v1
```

Comment:
This isn't needed. My guess is this was copied from the legacy example which it is needed there